package.json

```diff
@@ -27,6 +27,7 @@
         "node-int64": "^0.4.0",
         "rc-menu": "^7.4.21",
         "react": "^16.7.0",
+        "react-appinsights": "^3.0.0-rc.5",
```

Comment:
looks like they're close to releasing v3? Did you run into any weirdness w/ this RC?

Reply:
no, haven't notice anything strange